Flick/Networking/NetworkManager.swift

```diff
@@ -322,67 +328,76 @@ class NetworkManager {
         }
     }
 
-    /// [POST] Send invites to friends by usernames
-    static func sendFriendInvites(userId: String, usernames: [String], completion: @escaping ([String]) -> Void) {
+    /// [POST] Create a friend request [updated as of 1/1/21]
+    static func createFriendRequest(friendId: Int, completion: @escaping (Bool) -> Void) {
```

Comment:
this is just removing unused functions and moving the correct ones in this section